### DISCUSSION TITLE
Feature Request: Mass Edit Movies and Shows in Trailarr

### DISCUSSION THREAD

**Discussion by alexkahler** [nandyalu/trailarr#48] [Ideas]
Hey, I've been using Trailarr for a few days and found the process of unmonitoring movies to be quite tedious. Currently, you can only unmonitor movies one at a time—clicking into each movie, adjusting the monitor status, and then navigating back out. It's quite the rinse-and-repeat process, especially when dealing with a large number of movies.

I think Trailarr could really benefit from a feature allowing mass edits and updates to the status (including deletion) of multiple movies directly from the main screen. A great example of this functionality is found in Radarr and Sonarr:
![image](https://github.com/user-attachments/assets/95a10891-4838-4b3d-bdb5-a3e1ec87a9b8)

**Comment by nandyalu** [comment]
I do think this is needed but I need to find out how to do it in the frontend. I can add some logic on the backend portion fairly easily, but require some time to fully implement this.

**Comment by nandyalu** [comment]
This has been added in `v0.3.0`
